src/main/java/ch/uzh/ifi/hase/soprafs21/service/WebSocketService.java

```diff
@@ -179,4 +185,11 @@ public void sendGameSessionInvitation(UUID gameSessionId, String sessionIdentity
         sendToPlayer(sessionIdentityOfInvitedUser, path, DogUtils.generateGameSessoinInviteUserDTO(gameSessionId, hostName));
     }
 
+    public void broadcastThrowAway(UUID gameId, String playerName, List<String> cardCodes){
```

Comment:
würde entweder hier umbenennen zu sendThrowAway, weils immer so gemacht worde ist bis jetzt, oder alle die was in ein topic senden umbenennen zu broadcastXY, damit wir konsistent bleiben

Reply:
oke habe ich angepasst, jetzt neu broadcast zu topics und send zu queues

Reply:
heftig geil